styles/globals.css

```
@@ -33,3 +33,13 @@
 aside {
     top: calc(2rem + 80px);
 }
+
+.answer-list {
+    counter-reset: answer-list;
+}
+.answer-list .answer-list__item::before{
```

```suggestion
}

.answer-list .answer-list__item::before{
```

I think it's better to have a newline here?

---

Agree